sysutils/pfSense-pkg-apcupsd/files/usr/local/pkg/apcupsd.inc

```diff
@@ -42,6 +42,11 @@ function php_deinstall_apcupsd() {
 
 }
 
+function is_int_with_minus_one($arg) {
+  return (is_int($arg) || is_string($arg) && preg_match('/^-?\d+$/', trim($arg)))
+         && intval($arg) >= -1;
```

Comment:
This function can be simplified doing
```php
    return (is_int($arg) || $arg == '-1');
```

and since the result is really simple conditional I believe the function should be removed and the check done on both cases below.

Reply:
Your suggestion would

- return true for -2 (because it is an int)
- return false for "2" (because it is a string)

All existing validation seems to take strings and numbers, so I designed this function in this way too.
Also, -2 is not a valid value.

Reply:
Bumped PORTREVISION

Reply:
> Your suggestion would
> 
> * return true for -2 (because it is an int)
> * return false for "2" (because it is a string and not '-1')
> 
> All existing validation seems to take strings and numbers, so I designed this function in this way too.
> Also, -2 is not a valid value.

You can use `is_numericint()` from util.inc instead, it will make sure it's a positive int

Reply:
Done